api/app/controllers/mno_enterprise/jpi/v1/app_reviews_controller.rb

```diff
@@ -46,49 +42,64 @@ def update
     def destroy
       @app_review.destroy
       after_save
-
+      @app_review = nil
       render :show
```

Comment:
The frontend is using the apps average rating :( That is why it is uing a show. Which i not working if  you don't set app_review to nil, as the access to the fields are not possible, once destroy has been called.